readme.md

```diff
@@ -12,16 +12,29 @@ pip3 install keyboard
 ## How to use with Target Language(s)
 - For English only
 ```
-syntaxalert.py -words ./words/en.json
+python syntaxalert.py -words ./words/en.json
```

Comment:
is not required to add python to launch the script on unix because is already defined, is enough to give execution permission.
Can you remove them?